tools/bumper-bot/bumper.go

```diff
@@ -0,0 +1,104 @@
+//usr/bin/env go run $0 "$@"; exit
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/kubevirt/cluster-network-addons-operator/tools/bumper-bot/parser"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Error: must contain path to components yaml", os.Args)
+		return
+	}
+	componentsYaml := os.Args[1]
+
+	fmt.Println("parsing", componentsYaml)
+	components, err := parser.ParseComponentsYaml(componentsYaml)
+	if err != nil {
+		fmt.Println(errors.Wrap(err, "failed to parse components yaml"))
+		return
+	}
+
+	for i, component := range components {
```

Comment:
You can iterate maps with golang no need to transform components into slice.

Reply:
mmm, ComponentsConfig is not a map, it's a struct that holds structs. I'm thinking of using [this](https://stackoverflow.com/questions/18926303/iterate-through-the-fields-of-a-struct-in-go) to iterate.